source/reference/json/file-options-online-archive.txt

```diff
@@ -0,0 +1,99 @@
+.. _atlas-cli-file-options-online-archive-file:
+
+========================================
+Online Archive Configuration File
+========================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can use an Online Archive configuration file to specify the required 
+settings for creating and updating an Online Archive. This archive stores 
+data from one cluster within one project. To use this resource, the 
+requesting API Key must have the Project Data Access Admin role.
+
+Use the following resources to:
+
+- Learn the required settings you can specify in the Online Archive 
+  configuration file.
+
+- View and copy sample configuration files.
+
+Required Online Archive Create and Update Settings
+--------------------------------------------------
+
+|service| requires the following settings to create an Online Archive with
+the {+atlas-cli+}. You must specify these Online Archive settings either
+in the configuration file or as flags in the command:
+
+.. list-table:: 
+   :header-rows: 1 
+   :widths: 20 10 70 
+
+   * - Field 
+     - Type 
+     - Description 
+
+   * - ``groupId``
+     - string
+     - Unique identifier for your project. Your group id is the same 
+       as your project id. For existing groups, your group/project id 
+       remains the same. The resource and corresponding 
+       endpoints use the term groups.
+
+   * - ``archiveID``
+     - string
+     - Unique 24-hexadecimal digit string that identifies 
+       the online archive to update. This param is required for updating an
+       existing Online Archive.
```

Comment:
```suggestion
       the online archive to update.
```
Suggestion: I don't think the second sentence is needed since the first one states it's for updating an online archive.